### PR TITLE
feat(command): Add list-(un)synced-commands flags

### PR DESCRIPTION
**Pull request by badosu**

### CODE REVIEW

File: rts/Game/Game.cpp [hunk 1]
```diff
@@ -1646,7 +1646,7 @@ void CGame::StartPlaying()
 		eventHandler.GameStart();
 }
 
-static const char* const tracingSimFrameName = "SimFrame";
```
Comment (badosu) [inline rts/Game/Game.cpp:1649]: unused variable, can revert this if used in some way my LSP didn't find

File: rts/Game/Game.cpp [hunk 2]
```diff
@@ -1296,7 +1296,7 @@ bool CGame::UpdateUnsynced(const spring_time currentTime)
 
 			// if we have a new sim frame, then check when the time and CTO of the previous draw frame was. 
 			drawsimratio = std::fmin(drawsimratio, 1.0);  // Clamp it otherwise we will accumulate delay when < 30 FPS
-			float oldCTO = globalRendering->timeOffset;
```
Comment (badosu) [inline rts/Game/Game.cpp:1299]: unused variable, can revert this if used in some way my LSP didn't find

File: rts/Game/IGameCommands.h [hunk 3]
```diff
@@ -73,6 +77,28 @@ class IGameCommands
 		return sortedExecutors;
 	}
 
+	std::string JsonOutput() {
+		Json::Value root;
+
+		const auto actions = GetSortedActionExecutors();
+
+		for (const auto& pair: actions) {
```
Comment (lhog) [inline rts/Game/IGameCommands.h:85]: `for (const auto& [name, actionExecutor]: actions) {`

**Review by BAR-eL-bArTo — APPROVED** [review]
lgtm. This is some of the best, well written code I've had the privilege of laying eyes on.

**Comment by lhog** [comment]
We've entered the engine merge freeze stage, but this one can still go in when ready.

**Comment by badosu** [comment]
> lgtm. This is some of the best, well written code I've had the privilege of laying eyes on.

I feel honoured and privileged of such a patrician review.

**Pull request merged**